sparkly/__init__.py

```diff
@@ -19,4 +19,4 @@
 assert SparklySession
 
 
-__version__ = '2.3.0'
+__version__ = '2.4.0'
```

Comment:
wonder if we were trying to support some easy mapping between sparkly and pyspark, we did well so far but not with this change

Reply:
Hm I think it was a coincidence as it doesn't sound like a viable target. It would mean that we should replicate Spark's development cycle, which sounds kinda slow